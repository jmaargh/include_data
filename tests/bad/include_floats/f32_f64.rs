use include_data::include_floats;

fn main() {}

// Type mismatch
static BYTES_32_f32_f64: &[f32] = include_floats!(f64, "../../test_data/binary_32");
