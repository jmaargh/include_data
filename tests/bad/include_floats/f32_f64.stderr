error[E0308]: mismatched types
 --> tests/bad/include_floats/f32_f64.rs:6:35
  |
6 | static BYTES_32_f32_f64: &[f32] = include_floats!(f64, "../../test_data/binary_32");
  |                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected `&[f32]`, found `&[f64]`
  |
  = note: expected reference `&'static [f32]`
             found reference `&'static [f64]`
  = note: this error originates in the macro `include_floats` (in Nightly builds, run with -Z macro-backtrace for more info)
